src/rules/max-nested-describe.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import { createRule, isDescribeCall } from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Enforces a maximum depth to nested describe calls',
+      recommended: false,
+    },
+    messages: {
+      exceededMaxDepth:
+        'Too many nested describe calls ({{depth}}). Maximum allowed is {{max}}.',
```

Comment:
```suggestion
        'Too many nested describe calls ({{ depth }}). Maximum allowed is {{ max }}.',
```

nit: I typically put spaces around the names (and so most if not all of our rules are like this) - we don't have a lint rule for enforcing this, but would be nice to be consistent.